packages/snaps-webpack-plugin/src/plugin.test.ts

```diff
@@ -215,6 +220,17 @@ describe('SnapsWebpackPlugin', () => {
       sourceCode: expect.any(String),
       writeFileFn: expect.any(Function),
     });
+
+    const writeFileFn = mock.mock.calls[0][1]?.writeFileFn;
```

Comment:
I'm not sure I fully understand this change 🤔

Reply:
It gets the `writeFileFn` that `mock` (`checkManifest`) was called with, and calls that, so that we can assert that `writeManifest` is called. I had to add it for coverage purposes mainly.

Reply:
Shouldn't it already be called earlier in the test and we can assert that?

Reply:
No, as we mock `checkManifest`, `writeFileFn` is never called.